src/main/java/cpw/mods/ironchest/IronChestType.java

```diff
@@ -32,21 +31,21 @@
     GOLD(81, 9, true, "Gold Chest", "goldchest.png", 1, Arrays.asList("ingotGold"), TileEntityGoldChest.class, "mmmmPmmmm", "mGmG4GmGm"),
     DIAMOND(108, 12, true, "Diamond Chest", "diamondchest.png", 2, Arrays.asList("gemDiamond"), TileEntityDiamondChest.class, "GGGmPmGGG", "GGGG4Gmmm"),
     COPPER(45, 9, false, "Copper Chest", "copperchest.png", 3, Arrays.asList("ingotCopper"), TileEntityCopperChest.class, "mmmmCmmmm"),
-    SILVER(72, 9, false, "Silver Chest", "silverchest.png", 4, Arrays.asList("ingotSilver"), TileEntitySilverChest.class, "mmmm3mmmm", "mGmG0GmGm"),
+    STEEL(72, 9, false, "Steel Chest", "steelchest.png", 4, Arrays.asList("ingotSteel"), TileEntitySteelChest.class, "mmmm3mmmm", "mGmG0GmGm"),
```

Comment:
Changing the `modelTexture` will break existing texturepacks. Please revert.

Reply:
guess change only "Silver Chest" to "Steel Chest". This will not affect the texture and naming stuff

Reply:
Arrays.asList("ingotSteel"),  guess we dont want ingots in recips.

Reply:
I can revert the texture - the ingotSteel is the default vanilla recipe (which gets removed)

I can try merging those changes in too, but wanted to do one thing at a time.